caddyhttp/rewrite/to.go

```diff
@@ -12,8 +12,7 @@ import (
 
 // To attempts rewrite. It attempts to rewrite to first valid path
 // or the last path if none of the paths are valid.
-// Returns true if rewrite is successful and false otherwise.
-func To(fs http.FileSystem, r *http.Request, to string, replacer httpserver.Replacer) Result {
+func To(fs http.FileSystem, r *http.Request, to string, replacer httpserver.Replacer) (result Result) {
```

Comment:
I typically try to avoid named returns; I don't really know what we gain from this one here.